### PR TITLE
IN-1185 Adds azure-cloud-account & gcp-cloud-acocunt templates

### PR DESCRIPTION
**Pull request by Sufian22**
Related to IN-1185

### CODE REVIEW

File: instruqt-tracks/azure-cloud-account/01-explore-console/assignment.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+slug: explore-console
+id: tlfoadlgoey6
+type: challenge
+title: explore-console
+teaser: Explore the console
+notes:
+- type: text
+  contents: |-
+    An Azure Subscription has been created for you.
```
Comment (victorfrederiknielsen) [inline instruqt-tracks/azure-cloud-account/01-explore-console/assignment.md:10]: During the notes screen, the creation of resources is happening. I would just move this to the present tense.

>An Azure Subscription is being created for you...

File: instruqt-tracks/azure-cloud-account/track.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+id: sxyq1fdxnqwz
+type: track
+title: Azure Cloud Account
+teaser: This track includes an Azure Subscription with Contributor role associated.
+description: |-
+  This track shows you how to configure Instruqt with an Azure account with Contributor level privileges granted, so make sure you keep it private.
+  You can customize the permissions with the `roles` setting in `config.yaml` file.
+icon: https://storage.googleapis.com/instruqt-frontend/img/tracks/default.png
+tags: []
+owner: instruqt
```
Comment (victorfrederiknielsen) [inline instruqt-tracks/azure-cloud-account/track.yml:11]: The owner should be `templates` as it refers to the organization we're pushing to.

File: instruqt-tracks/azure-cloud-account/01-explore-console/assignment.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+slug: explore-console
+id: tlfoadlgoey6
```
Comment (victorfrederiknielsen) [inline instruqt-tracks/azure-cloud-account/01-explore-console/assignment.md:3]: For this repository, we should remove id's, this way the track is environment independent. We can then use the CLI to push to either dev or prod with the `--env dev` flag. You can check out the logic in the Workflows.

We can still identify the track with the Track slugs, which should be identical.

File: instruqt-tracks/gcp-cloud-account/01-explore-console/assignment.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+slug: explore-console
+id: 4urtpmgcxmrm
```
Comment (victorfrederiknielsen) [inline instruqt-tracks/gcp-cloud-account/01-explore-console/assignment.md:3]: Also with assignments, remove the ids

File: instruqt-tracks/azure-cloud-account/config.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+version: "3"
+containers:
+- name: cloud-client
+  image: gcr.io/instruqt/cloud-client
+  ports: [80]
+  shell: /bin/bash
+azure_subscriptions:
+- name: azuresubscription
+  roles:
+  - Contributor
```
Comment (victorfrederiknielsen) [inline instruqt-tracks/azure-cloud-account/config.yml:10]: We should be consistent with the formatting of the files. I think Prettier should fix everything up, if you're not using it already. Specifically, I am referring to the "end of file newline" rule, which should be present.

We should add a code style guide to this repo 💡

File: instruqt-tracks/gcp-cloud-account/track.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+title: GCP Account
+teaser: This track includes a Google Cloud Project with access to Compute service.
+description: |-
+  This track shows you how to configure Instruqt with a GCP account, it grants administrator level privileges to Compute service so make sure you keep it private.
+  You can customize the permissions with the `services` setting in `config.yaml` file.
+icon: https://storage.googleapis.com/instruqt-frontend/img/tracks/default.png
+tags:
+- cloud
+- gcp
+owner: instruqt
```
Comment (victorfrederiknielsen) [inline instruqt-tracks/gcp-cloud-account/track.yml:13]: Same, should be `templates`

**Comment by victorfrederiknielsen** [comment]
Note: For better observability, we should add a short description to this PR. It should describe what we're adding to the repository. You can use these as a reference. https://github.com/instruqt/frontend/pull/1263 or https://github.com/instruqt/frontend/pull/1267.

This is not a rule we're working with, but something that I've personally had help.

For the title of the PR, we should prefix it with the ticket number as such:

> IN-1185 Adds `azure-cloud-account` & `gcp-cloud-acocunt` templates

Although I do really like the idea of using `feat:`/`fix:` "conventional" commit messages. Let's discuss as a team!

**Comment by Sufian22** [comment]
I've already fixed the review changes on [866c1fc](https://github.com/instruqt/templates/pull/30/commits/866c1fcab837c32f847c7ca3c2b57903d6406775).

Yeah, it would be great to have a common convention to have a clean git log on the main branches 🔥

**Review by victorfrederiknielsen — APPROVED** [review]
Great work Sufian! 🚀 🚀

**Pull request merged**